alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/patch.sh

```diff
@@ -15,6 +15,9 @@ find "${GENERATED_SOURCE_DIR}" -type f -name "*.java" -exec sed \
   -e 's;url = "${processServicesEnterprise_.url.*}";url = "${process.service.url}", path = "${process.service.path}";g' \
   -e 's;url = "${alfresco-activiti-deployment-rest-api.url.*}";url = "${alfresco.service.deployment.url}", path = "${alfresco.service.deployment.path}";g' \
   -e 's;url = "${alfresco-activiti-modeling-rest-api.*}";url = "${modeling.url}", path = "${modeling.path}";g' \
+  -e 's;\(value = "\)/\(.*, url\);\1\2;g' \
+  -e 's;\(value = "\)\(.*\)-\(.*\, url\);\1\2\3;g' \
```

Comment:
Removes `-` present in some `vendorExtensions.x-service-url-prefix` that makes `@FeignClient` value invalid.